third_party/bazel/mkl/mkl_configure.bzl

```diff
@@ -0,0 +1,115 @@
+"""Setup TensorFlow as external dependency"""
+
+_MKL_ROOT = "MKL_ROOT"
+_WITH_MKL = "BLADE_WITH_MKL"
+
+_MKL_STATIC_LIBS = [
+    "mkl/lib/intel64_lin/libmkl_intel_ilp64.a",
+    "mkl/lib/intel64_lin/libmkl_intel_thread.a",
+    "mkl/lib/intel64_lin/libmkl_core.a",
+    "compiler/lib/intel64_lin/libiomp5.a",
+]
+
+_MKL_IOMP5_SO = "compiler/lib/intel64_lin/libiomp5.so"
+
+def _tpl(repository_ctx, tpl, substitutions):
+    repository_ctx.template(
+        tpl,
+        Label("//bazel/mkl:%s.tpl" % tpl),
+        substitutions,
+    )
+
+def _find_recursive(repository_ctx, dir):
+    result = repository_ctx.execute(["find", "-L", dir, "-type", "f"])
+    if result.return_code != 0:
+        fail("Repository command failed: \n %s" % result.stderr.strip())
+    return result.stdout.strip().splitlines()
+
+def _cc_import(repository_ctx, lib_path):
+    fname = repository_ctx.path(lib_path).basename
+    return """
+cc_import(
+    name = "{}",
+    static_library = "{}",
+    alwayslink = 0,
+)""".format(fname, lib_path)
+
+def _genrule_copy(repository_ctx, name, srcs, outs):
```

Comment:
Maybe it's time to remove all these `genrule`s to make it simpler, just like what we do here: https://github.com/alibaba/BladeDISC/blob/f2b15f1aa2829450635b33caf016efc8c2d38a2b/pytorch_blade/bazel/torch/repo.bzl#L8

Reply:
done